core/src/main/java/com/rim/IceField/Game.java

```diff
@@ -173,6 +172,7 @@ public void newGame (ArrayList < PlayerBase > playersList, ArrayList < String >
                 if (randomBlow[currentRound]) {
                     Blizzard.blow(players, map);
```

Comment:
Since we can send the list of inputs to the UserInteraction, we can use the same function. There is no need to write this twice. Remember the DRY principle Don't Repeat Yourself